Tests/NIOTests/EventLoopTest.swift

```diff
@@ -91,6 +91,64 @@ public class EventLoopTest : XCTestCase {
         XCTAssertFalse(ran.load())
     }
 
+    public func testScheduleRepeatedTask() throws {
+        let nanos = DispatchTime.now().uptimeNanoseconds
+        let initialDelay: TimeAmount = .milliseconds(50)
+        let delay: TimeAmount = .milliseconds(100)
```

Comment:
can we maybe make this `5` and `10` milliseconds? Otherwise this test alone will run almost 1s which is quite slow